### DISCUSSION TITLE
Customizing Alert Type Color

### DISCUSSION THREAD

**Discussion by gianfrancopiana** [themesberg/flowbite-vue#221] [Q&A] [answered]
I want to use my own colors for Alert component types like "danger", "info", "success" and so on. But it seems these colors are hardcoded Tailwind CSS classes.

Two potential options I see:
	1.	Override the Tailwind CSS classes in my project's stylesheet.
	2.	Modify the component source code by forking the library to accept colors.
	
These either seem workaround or come with an overhead. Is there an existing or built-in way to customize these ﻿type colors? If not, what's the recommended approach for this use case?

Thanks for the help.

**Comment by cogor** [comment, marked as answer]
Hey, there is currently no convenient way to override colors for each type, but there are plans to add a global configuration where this can be done.
Right now you can use this method, it's not very pretty, but it's better than global color override
https://stackblitz.com/edit/vitejs-vite-hvdkgb?file=src%2Fcomponents%2FMyAlert.vue
